src/Behat/Behat/Output/Node/Printer/ListPrinter.php

```diff
@@ -285,12 +285,12 @@ private function getLocationFromScope(?HookScope $scope): ?string
                 ),
                 $scope instanceof BeforeScenarioScope,
                 $scope instanceof AfterScenarioScope => $this->configurablePathPrinter->processPathsInText(
-                    $scope->getFeature()->getFile()
-                ) . ':' . $scope->getScenario()->getLine(),
+                    $scope->getFeature()->getFile() . ':' . $scope->getScenario()->getLine()
```

Comment:
The path printer now expects to find the line number as part of the text so in several places we add it to the text before sending it to printer instead of adding it to the returned text

Reply:
It looks like this is just a code formatting change, unless I'm missing something?

Reply:
if you look carefully you will see that in the previous code the concatenation of the line number happens after the call to `processPathsInText` and now it happens before.

Reply:
Of course, sorry :facepalm: